[error-tracking] Improve monitor documentation (#28452)
DataDog/documentation: content/en/monitors/types/error_tracking.md

There are two types of alerting conditions you can configure your Error Tracking monitor with:

| Alerting&nbsp;condition     | Description    |
| ---  | ----------- |
|New Issue| Alert when an issue occurs for the first time or a regression occurs. For example, alert for your service whenever more than 2 users are impacted by a new error. |
|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |

### Define alert conditions

{{< tabs >}}

{{% tab "New Issue" %}}
#### Issues to alert on

> New issue monitors alert on issues that are **For Review** and that meet your alerting conditions. Read more about [Issue States here][1]. As regressions are transitioned to **For Review** automatically, they are automatically monitored with New Issue monitors.

Reviewer: ```suggestion
New issue monitors alert on issues that are in the **For Review** state and meet your alerting conditions. Regressions are automatically transitioned to the For Review state, so they are monitored by default with New Issue monitors. For more information on states, see [Issue States][1].
```

Select **All**, **Browser**, **Mobile**, or **Backend** issues and construct a search query using the same logic as the [Error Tracking Explorer search][2] for the issues' error occurrences.

> <div class="alert alert-info"><strong>Note</strong>: New Issue monitors only consider issues that were created or regressed after the monitor was created or last edited, with a 24-hour lookback period.</div>

Reviewer: Recommend removing Note. There's an implied **Note** in an alert div.

```suggestion
<div class="alert alert-info">New Issue monitors only consider issues that were created or regressed after the monitor was created or last edited. These monitors have a 24-hour lookback period.</div>
```

#### Define alert threshold

Choose one of the following options:

> **Alert on all new issues**:

Reviewer: This is optional, but from the Staging preview, it's difficult to distinguish the header and the options. Putting the two options inside a collapse content makes it a little clearer on what actions the reader can take.

```suggestion
{{% collapse-content title="Alert on all new issues" level="p" %}}

```

> Monitor triggers when any new issue is detected (the number of errors is greater than 0 over the past day).

Reviewer: ```suggestion
Monitor triggers when any new issue is detected (the number of errors is greater than 0 over the past day).

{{% /collapse-content %}}

```

> Or **define your alert metric**:

Reviewer: ```suggestion
{{% collapse-content title="Define your alert metric" level="p" %}}
```

1. Choose the metric you want to monitor. There are three suggested filter options to access the most frequently used facets:

    - **Error Occurrences**: Triggers when the error count is `above`.
    - **Impacted Users**: Triggers when the number of impacted user emails is `above`.
    - **Impacted Sessions**: Triggers when the number of impacted session IDs is `above`.

    If you select **All** or **Backend** issues, only the **Error Occurrences** option is available.

    You can also specify a custom measure you want to use to monitor. If you select a custom measure, the monitor alerts when the count of unique values of the facet is `above`.

> 2. Have a notification for each issue that matches your query, and group the results by any other attribute you require (e.g. have a notification for each issue matching the query, and on each environment).

Reviewer: ```suggestion
2. Have a notification for each issue that matches your query, and group the results by any other attribute you require (for example, have a notification for each issue matching the query, and on each environment).
```

3. Query data over the last day (by default) or any other time window at each evaluation.

4. Choose a threshold for the monitor to trigger (by default 0, i.e. at the first occurrence).

#### Programmatic management

If you are using Terraform or custom scripts using our public APIs to manage your monitors, you need to specify some clauses in the monitor query:

> * Add the source you want to target between **All**, **Browser**, **Mobile**, and **Backend** issue. Use the `.source()` clause with `"all"`, `"browser"`, `"mobile"` or `"backend"` right after your filter. Note you can only use one at a time.

Reviewer: ```suggestion
* Add the source you want to target between **All**, **Browser**, **Mobile**, and **Backend** issue. Use the `.source()` clause with `"all"`, `"browser"`, `"mobile"` or `"backend"` right after your filter. **Note**: you can only use one at a time.
```

[2]: /error_tracking/explorer
[3]: /monitors/configuration/#alert-grouping/
{{% /tab %}}

{{% tab "High Impact" %}}
#### Issues to alert on

High Impact monitors alert on issues that are **For Review** or **Reviewed** and that meet your alerting conditions. Read more about [Issue States][1].

Select **All**, **Browser**, **Mobile**, or **Backend** issues and construct a search query using the same logic as the [Error Tracking Explorer search][2] for the issues' error occurrences.

#### Define alert threshold
1. Choose the metric you want to monitor. There are three suggested filter options to access the most frequently used facets:

    - **Error Occurrences**: Triggers when the error count is `above`.
    - **Impacted Users**: Triggers when the number of impacted user emails is `above`.
    - **Impacted Sessions**: Triggers when the number of impacted session IDs is `above`.

    If you select **All** or **Backend** issues, only the **Error Occurrences** option is available.

    You can also specify a custom measure you want to use to monitor. If you select a custom measure, the monitor alerts when the count of unique values of the facet is `above`.

> 2. Have a notification for each issue that matches your query, and group the results by any other attribute you require (e.g. have a notification for each issue matching the query, on each environment).

Reviewer: ```suggestion
2. Have a notification for each issue that matches your query, and group the results by any other attribute you require (For example, have a notification for each issue matching the query, on each environment).
```

3. Query data over the last day (by default) or any other time window at each evaluation.

> 4. Choose a threshold for the monitor to trigger (by default 0, i.e. at the first occurrence).

Reviewer: ```suggestion
4. Choose a threshold for the monitor to trigger (by default 0-triggers at the first occurrence).
```

#### Programmatic management

If you are using Terraform or custom scripts using our public APIs to manage your monitors, you need to specify some clauses in the monitor query:

> * Add the source you want to target between **All**, **Browser**, **Mobile**, and **Backend** issue. Use the `.source()` clause with `"all"`, `"browser"`, `"mobile"` or `"backend"` right after your filter. Note you can only use one at a time.

Reviewer: ```suggestion
* Add the source you want to target between **All**, **Browser**, **Mobile**, and **Backend** issue. Use the `.source()` clause with `"all"`, `"browser"`, `"mobile"` or `"backend"` right after your filter. **Note**: you can only use one at a time.
```
